src/MartinGeorgiev/Doctrine/ORM/Query/AST/Functions/StrConcat.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MartinGeorgiev\Doctrine\ORM\Query\AST\Functions;
+
+/**
+ * Implementation of PostgreSQL string concatenation (using ||).
+ *
+ * @see https://www.postgresql.org/docs/current/functions-string.html
+ * @since 2.6.0
+ *
+ * @author Alexander Makarov <sam@rmcreative.ru>
+ */
+class StrConcat extends BaseFunction
```

Comment:
```suggestion
class StrConCat extends BaseFunction
```

Reply:
I disagree. "concatenation" is a single word.